sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1204,10 +1204,14 @@ export class ApplicationService extends RecordDataModelService<Application> {
         "programYear.startDate",
         "user.userName",
```

Comment:
Can you please make sure that required extensions to VS code are installed, settings.json is updated(any dev can share that) and also the EOL git command is run?